definitions/vertical_scaling.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Vertical Scaling
+status: Completed
+category: Concept
+---
+
+## Vertical Scaling
+
+### What it is
+
+The ability for an application to scale, typically by adding CPU and memory as the workload increases. Also known as “scale up”, this refers to adding resources to a single node.
```

Comment:
@JasonMorgan, I think we should mirror the horizontal scaling definition a little. Here's my suggestion:

Vertical scaling, also known as "scaling up and down", is a technique where a system's capacity is increased by adding CPU and memory to individual [nodes](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) as the workload increases. Let's say, you have a computer of 4GB RAM and want to increase its capacity to 16GB RAM, scaling it vertically means switching to a 16GB RAM system. (Please refer to [horizontal scaling](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md) for a different scaling approach.)